jar.go

```diff
@@ -252,6 +324,8 @@ func (j *Jar) setCookies(u *url.URL, cookies []*http.Cookie, now time.Time) {
 		return
 	}
 	if u.Scheme != "http" && u.Scheme != "https" {
+		// TODO is this really correct? It might be nice to send
```

Comment:
The answer, as far as I can tell is its up to us to decide the policy. I think it would be useful to support websocket. I think we need to keep the cookies separate though.